pkg/cluster/clusterserviceprincipal_test.go

```diff
@@ -297,9 +301,13 @@ func getFakeOpenShiftSecret() corev1.Secret {
 	name := "azure-credentials"
 	namespace := "kube-system"
 	data := map[string][]byte{
```

Comment:
`updateOpenShiftSecret` now sets these fields, so they need to be present on the fake object.